ParameterDataApi.yaml

```diff
@@ -18,13 +18,13 @@ tags:
   description: Parameter data model manipulation endpoints
 
 paths:
-  /parameter:
+  /timeSeries:
```

Comment:
Is there interest in maintaining the 'parameter' endpoint, since this is the 'parameter data api'?
This could easily be:

/parameter/series
/parameter/sample

to capture the two endpoints